gremlin-python/src/main/jython/gremlin_python/driver/remote_connection.py

```diff
@@ -42,7 +42,7 @@ def traversal_source(self):
 
     @abc.abstractmethod
     def submit(self, bytecode):
-        print "sending " + bytecode + " to GremlinServer..."
+        print("sending " + bytecode + " to GremlinServer...")
```

Comment:
No, typically I would use Python's `logging` module. As the above is part of an abstract class and never gets called, I left is as such. In the future we can set up some nice logging for gremlin-python.